plugins/woocommerce/src/Admin/WCAdminHelper.php

```diff
@@ -96,4 +96,26 @@ public static function is_wc_admin_active_in_date_range( $range, $custom_start =
 		}
 		return false;
 	}
+
+	/**
+	 * Test if the site is fresh. A fresh site must meet the following requirements.
+	 *
+	 * - The current user was registered less than 1 month ago.
+	 * - fresh_site option must be 1
+	 *
+	 * @return bool
+	 * @throws \Exception
+	 */
+	public static function is_site_fresh() {
+		$current_userdata = get_userdata( get_current_user_id() );
+		// Return false if we can't get user meta data for some reason.
+		if ( ! $current_userdata ) {
+			return false;
+		}
+
+		$date = new \DateTime( $current_userdata->user_registered );
+		$month_ago = new \DateTime( '-1 month' );
+
+		return get_option( 'fresh_site' ) === '1' && $date > $month_ago;
```

Comment:
I think we can make the code more efficient by moving the `fresh_site ` option check to the beginning of the function and early return the false if the option is 0. This could also improve readability. What do you think?

Reply:
That's a good suggestion! I'll make the change.

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/45232/commits/27d4faa30c1fdd870fdf21b51911ce876fb1eacb